cmd/tuf/app/init.go

```diff
@@ -108,18 +109,30 @@ func InitCmd(ctx context.Context, directory, previous string, targets targetsFla
 				return err
 			}
 		}
-		if err := repo.SetThreshold("root", threshold); err != nil {
+		if err := repo.SetThreshold(role, threshold); err != nil {
 			return err
 		}
 	}
 
-	// Add online snapshot and timestamp keys with a shorter expiration.
+	// Revoke old root keys used for snapshot and timestamp.
+	for _, role := range []string{"snapshot", "timestamp"} {
```

Comment:
The function of this FOR loop is not obvious, especially with the `tufKey.IDs()[0]`. Would you mind documenting a bit? Thanks!